### ISSUE TITLE
Incorrect UDP port when generating new user profile

### ISSUE THREAD

**Issue by richardgirges** [dcos-labs/dcos-openvpn#13]
Hey there. Thanks for this nifty package, it's doing wonders for us!

We came across a minor issue with the incorrect port being outputted in the OVPN profile when creating a new user.

### Repro
Create a new user
```
curl -k -u user:pass -X POST -d "name=john" https://<IP_ADDRESS_OMITTED>:5000/client
```

### Result
This is the beginning output of the successful POST request. The `5000 udp` should be `1194 udp`.
```
client
nobind
dev tun
remote-cert-tls server

remote <IP ADDRESS OMITTED> 5000 udp
```

### Expected Result
```
client
nobind
dev tun
remote-cert-tls server

remote <IP ADDRESS OMITTED> 1194 udp
```

We are currently fixing the ovpn profiles manually on our end after generating new VPN users.

**Comment by aggress** [comment]
@richardgirges apologies for the delay in response, I didn't get alerted of this issue being opened.  I'll take a look at it today for you.

Regards,
R

**Comment by aggress** [comment]
Fixed in 0.0.0-2.0 now merged into master and is live on the DC/OS Universe/Catalog
